lib/client.js

```diff
@@ -299,6 +299,18 @@ function isOriginValid (origin) {
   return false
 }
 
+function clearIframe (origin) {
```

Comment:
⛏ this could be split into two functions `clearDocument` and `appendTextToBody` (something along this line).

Reply:
Done!